jsonnet/kube-prometheus/components/grafana.libsonnet

```
@@ -44,7 +43,7 @@ function(params) {
         grafana: g._config.version,
       },
       imageRepos+:: {
-        grafana: g._config.imageRepos,
+        grafana: std.split(g._config.image, ":")[0],
```

```suggestion
        grafana: std.split(g._config.image, ':')[0],
```

jsonnet fmt doesn't like double quotes :shrug: